db/mysql/db.go

```diff
@@ -325,6 +325,11 @@ func (db *mysqlDB) Delete(ctx context.Context, table string, key string) error {
 	return db.execQuery(ctx, query, key)
 }
 
+func (db *mysqlDB) Analyze(ctx context.Context, table string) error {
+	_, err := db.db.Exec(fmt.Sprintf(`ANALYZE TABLE %s`, table))
```

Comment:
Does it depends on the `db` has executed `use db` or the `db` is created with database?

Reply:
we create the Go Database with the specific database before, so no need here.